plutus-ledger/src/Ledger/Typed/Scripts/Validators.hs

```diff
@@ -1,153 +1,7 @@
-    -- we can do this safely because the on-chain validators are untyped, so they always
-    -- take 'BuiltinData' arguments. The validator script stays the same, so the conversion
-    -- from 'BuiltinData' to 'a' still takes place, even if it's not reflected in the type
-    -- signature anymore.
-    TypedValidator{tvValidator, tvValidatorHash, tvForwardingMPS, tvForwardingMPSHash}
-
--- | Make a 'TypedValidator' from the 'CompiledCode' of a validator script and its wrapper.
-mkTypedValidator ::
-    CompiledCode (ValidatorType a)
-    -- ^ Validator script (compiled)
-    -> CompiledCode (ValidatorType a -> UntypedValidator)
-    -- ^ A wrapper for the compiled validator
-    -> TypedValidator a
-mkTypedValidator vc wrapper =
-    let val = Scripts.mkValidatorScript $ wrapper `applyCode` vc
-        hsh = ScriptUtils.validatorHash val
-        mps = MPS.mkForwardingMintingPolicy hsh
-    in TypedValidator
-        { tvValidator         = val
-        , tvValidatorHash     = hsh
-        , tvForwardingMPS     = mps
-        , tvForwardingMPSHash = ScriptUtils.mintingPolicyHash mps
-        }
-
--- | Make a 'TypedValidator' from the 'CompiledCode' of a parameterized validator script and its wrapper.
-mkTypedValidatorParam
-    :: forall a param. Lift DefaultUni param
-    => CompiledCode (param -> ValidatorType a)
-    -- ^ Validator script (compiled)
-    -> CompiledCode (ValidatorType a -> UntypedValidator)
-    -- ^ A wrapper for the compiled validator
-    -> param
-    -- ^ The extra paramater for the validator script
-    -> TypedValidator a
-mkTypedValidatorParam vc wrapper param =
-    mkTypedValidator (vc `PlutusTx.applyCode` PlutusTx.liftCode param) wrapper
-
--- | The hash of the validator.
-validatorHash :: TypedValidator a -> Scripts.ValidatorHash
-validatorHash = tvValidatorHash
-
--- | The address of the validator.
-validatorAddress :: TypedValidator a -> Address
-validatorAddress = scriptHashAddress . tvValidatorHash
-
--- | The validator script itself.
-validatorScript :: TypedValidator a -> Scripts.Validator
-validatorScript = tvValidator
-
--- | Make a 'TypedValidator' (with no type constraints) from an untyped 'Validator' script.
-unsafeMkTypedValidator :: Scripts.Validator -> TypedValidator Any
-unsafeMkTypedValidator vl =
-    let vh = ScriptUtils.validatorHash vl
-        mps = MPS.mkForwardingMintingPolicy vh
-    in
-    TypedValidator
-        { tvValidator         = vl
-        , tvValidatorHash     = vh
-        , tvForwardingMPS     = mps
-        , tvForwardingMPSHash = ScriptUtils.mintingPolicyHash mps
-        }
-
--- | The minting policy that forwards all checks to the instance's
---   validator
-forwardingMintingPolicy :: TypedValidator a -> Scripts.MintingPolicy
-forwardingMintingPolicy = tvForwardingMPS
+  {-# DEPRECATED "Use Plutus.Script.Utils.V1.Typed.Scripts.Validators instead" #-}
+  ( module Plutus.Script.Utils.V1.Typed.Scripts.Validators,
+  )
+where
```

Comment:
Nice reflex of using deprecation :)

Reply:
Which triggered warnings in our other packages, which made CI fail :D